packages/solutions/app-tools/src/analyze/generateCode.ts

```diff
@@ -285,9 +285,31 @@ export const generateCode = async (
 
       // generate entry file.
       if (config.source.enableAsyncEntry) {
+        let rawAsyncEntryCode = `import('./${ENTRY_BOOTSTRAP_FILE_NAME}');`;
+        const ssr = getEntryOptions(
+          entryName,
+          config.server.ssr,
+          config.server.ssrByEntries,
+          packageName,
+        );
+        if (ssr) {
+          rawAsyncEntryCode = `
+          export const serverRender = async (...args) => {
+            let exports = await ${rawAsyncEntryCode};
```

Comment:
`exports` is a special object that is used in Node.js modules, maybe we can use other variable names.